Adding method to ad busses to elements dict

FelixMau opened the pull request:
Adding method that takes finished dict with scalar elements and searches for bus entries within. Then adds the bus Dataframe to Dict as `"bus": pandas.DataFrame`

FelixMau committed:
Adding Timeseries function

FelixMau committed:
Calling ad busses to elements in build datapackage

FelixMau commented on data_adapter_oemof/build_datapackage.py:
```diff
@@ -63,6 +63,28 @@ def refactor_timeseries(timeseries: pd.DataFrame):
+    """
+    unique_entries = set()
+    for element in elements_dict.values():
+        bus_occourences = [col for col in element.columns if "bus" in col]
```
Do we have to use a different approach to find all columns containing the busses?

henhuy commented:
My idea would be to gather all busses while iterating over processes...
So start with an empty list and add them successively

FelixMau commented:
Will change to that.
For justification: I didn't want to extend the loop with to much functionality and keep the bus csv in a separate function.

FelixMau committed:
Adding bus to parametrized elements iterative

Adding busses by calling get_bus from Mapper. Also removed `cls` parameter from get_default and get bus since the class is stored in Mapper as `adapter` anyway.

FelixMau committed:
Cleaning

FelixMau committed:
Remove old function

FelixMau committed:
Learned about walrus (thx)

FelixMau committed:
Bus for each row.

FelixMau committed:
Merge pull request #35 from sedos-project/add_bus_elements

Adding bus to parametrized elements iterative
